python/revng/pypeline/pipeline_parser.py

```diff
@@ -55,9 +55,16 @@ def parse_pipe(
             )
         bindings.append(container_decls[arg])
 
-    configuration = task.get("configuration", "")
+    configuration = task.get("configuration")
+    if configuration is None:
+        configuration_string = ""
+    elif isinstance(configuration, str):
+        configuration_string = configuration
+    else:
+        configuration_string = yaml.safe_dump(configuration)
```

Comment:
Shall we use JSON? This serialization is almost never exposed to humans, it'd make sense to relax pipes to have JSON parser instead of a YAML parser. What do you think=